justfile

```diff
@@ -39,7 +39,7 @@ lint: lock
     {{uv_run}} ruff format --check --diff {{all}}
 
 # Run static type checks
-static *args: lock
+type *args: lock
```

Comment:
[non-block/foresight]: should it be `static` or `type`? Some of our repositories use `static` while others use `type`. We're invoking a "static type" checker, so it's not immediately clear to me which one we should pick, but we should pick one.

Personally I'm leaning towards `static` since we're checking the static types and annotations we've added to the code that are known before "compiling" the Python code at runtime. `type`, at least to me, is slightly more vague since it's not immediate clear what is being checked (dynamic v.s. static types).

We can bikeshed on this next week though since it's a shorter pulse.

Reply:
I took the name from the slurm-charms repo, but both `static` and `type` seem too vague IMO. I was considering changing it to `typecheck` which is a bit longer but easier to remember.